lib/Extractor/Candidates.cpp

```diff
@@ -151,7 +156,10 @@ Inst *ExprBuilder::makeArrayRead(Value *V) {
   if (Opts.NamedArrays)
     Name = V->getName();
   unsigned Width = DL.getTypeSizeInBits(V->getType());
-  return IC.createVar(Width, Name);
+  APInt KnownZero(Width, 0, false), KnownOne(Width, 0, false);
+  if (HarvestKnownBits)
+    computeKnownBits(V, KnownZero, KnownOne, DL, 0, 0);
```

Comment:
it looks like the last 2 arguments have the same default as the value you are passing, so I'd say leave them out